vivo-project/Vitro
# [VIVO-1839] - Use language name for selector instead of flags

**[JIRA Issue](https://jira.duraspace.org/projects/VIVO)**: https://jira.lyrasis.org/browse/VIVO-1839

* Other Relevant Links (Mailing list discussion, related pull requests, etc.)
From Slack 
<img width="1004" alt="Screen Shot 2020-05-21 at 1 08 38 PM" src="https://user-images.githubusercontent.com/10748475/82596246-39fae400-9b64-11ea-948f-f92f4db9465d.png">


# What does this pull request do?
Changes the language selector dropdown from displaying flag images to displaying the locale name.

<img width="267" alt="Screen Shot 2020-05-21 at 12 56 37 PM" src="https://user-images.githubusercontent.com/10748475/82597006-885cb280-9b65-11ea-8259-7f0570604616.png"> <img width="311" alt="Screen Shot 2020-05-21 at 12 56 52 PM" src="https://user-images.githubusercontent.com/10748475/82597001-8692ef00-9b65-11ea-89a6-babb785fb8f7.png">

# What's new?
Displays the locale name in the language selector (in that locale's language, e.g. the text for German will always say `Deutsch` regardless of which language is currently selected. 

# How should this be tested?
Build with changes. Confirm language selector still works as intended and languages are display as text, not images. 

# Additional Notes:
The label is generated from Java's Locale library, so the language must use a recognized locale code to work correctly. This matches the current behavior, which only made an exception for a special 'es-GO' translation that used to be included with VIVO (see https://github.com/vivo-project/Vitro/blob/6d64d27fe4ad26bee7ffd10a852e71240f8ae903/api/src/main/java/edu/cornell/mannlib/vitro/webapp/i18n/selection/LocaleSelectionSetup.java#L143). 

# Interested parties
@awoods
